src/object/obj_tx.c

```diff
@@ -1691,7 +1712,12 @@ dc_tx_cpd_sub_reqs_size(struct daos_cpd_sub_req *dcsr, int count)
 				size += sizeof(*oia->oia_offs) * oia->oia_iod_nr;
 
 			if (dcu->dcu_flags & ORF_CPD_BULK)
-				size += sizeof(*dcu->dcu_bulks) * dcsr->dcsr_nr;
+				/*
+				 * NOTE: In object layer, we cannot exactly know how large the
+				 *	 bulk handle will be in lower layer network (mercury).
+				 *	 So here, we give some estimation for that.
+				 */
+				size += 187 * dcsr->dcsr_nr;
```

Comment:
can you please explain where 187 came from?
it's also better if we define it in some internal header and document for others to have some idea what this means.

Reply:
I wil update the comment.